### ISSUE TITLE
Change 'inf' label in legend

### ISSUE THREAD

**Issue by sampr7** [python-windrose/windrose#158]
_inf_ label in the legend is not appropriate on most of the occasions. Instead of that, a greater than (>) symbol can be added to the maximum value.

legend labels
[0.0 : 2.0 )
[2.0 : 4.0 )
[4.0 : 6.0 )
[6.0 : inf )

from this to this
[0.0 : 2.0 )
[2.0 : 4.0 )
[4.0 : 6.0 )
\>6.0

**Comment by sampr7** [comment]
Change 
https://github.com/python-windrose/windrose/blob/b602213d9d6250d9e6eebaa7ac992fe9b6ec1b7a/windrose/windrose.py#L215-L230

to
```
        def get_labels(decimal_places=1, units=None):
            _decimal_places = str(decimal_places)

            fmt = "[%." + _decimal_places + "f " + ": %0." + _decimal_places + "f"

            labels = np.copy(self._info["bins"])
            labels[-1] = np.nan
            last_dig = labels[-2]
            if locale.getlocale()[0] in ["fr_FR"]:
                fmt += "["
            else:
                fmt += ")"

            if units:
                fmt += ' ' + units

            labels = [fmt % (labels[i], labels[i + 1]) for i in range(len(labels) - 1)]
            print(labels)
            labels[-1] = '>'+str(last_dig)
            return labels
```
![windrose2](https://user-images.githubusercontent.com/65779808/133761235-4da074d6-4563-49f7-a313-67da6e5129eb.png)

**Comment by evanleeturner** [comment]
I am having this issue as well.  Is there a plan to introduce this fix for the windrose legend?

**Comment by ocefpaf** [comment]
@sampr7 your fix looks OK to me. Do you want to send a PR?

**Comment by sampr7** [comment]
> @sampr7 your fix looks OK to me. Do you want to send a PR?

@ocefpaf Sure. Will do

**Comment by ocefpaf** [comment]
Thanks @sampr7! We merged #185 and I'll prepare a new release soon.
